stream-chat-android-ui-components/src/main/res/values/attrs.xml

```diff
@@ -127,6 +127,69 @@
         <attr name="streamUiAddChannelBackgroundTint" format="color|reference" />
     </declare-styleable>
 
+    <!--ChannelListView-->
+    <declare-styleable name="ChannelListView">
```

Comment:
We were previously reusing the styleable declaration in the old UI module here, this duplicates the styleable instead. We should still review if all of these are hooked up in the relevant style class, and implemented in the View though. We might have to remove some of these, if we don't implement them in the new UI implementation.